pkg/distro/rhel/imagetype.go

```diff
@@ -216,6 +216,10 @@ func (t *ImageType) getDefaultInstallerConfig() (*distro.InstallerConfig, error)
 }
 
 func (t *ImageType) PartitionType() string {
+	if t.BasePartitionTables == nil {
+		return ""
```

Comment:
This is just food for thought, but it made me wonder if we should not define `PartitionType` as an enum and have an explicit value for `None` instead of an empty string. Comparing enums is IMHO less error-prone than comparing strings against literal string values.

Reply:
We have a type like this https://github.com/osbuild/images/pull/1004/files now

Reply:
I opened https://github.com/osbuild/images/pull/1025 ?

Reply:
Sweet, thanks!